CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+## Running locally
+Start local site with:
+
+```shell
+wp-env start
+```
+
+Change permalink structure:
+
+```shell
+wp-env run cli "wp rewrite structure '/%postname%'"
```

Comment:
This is a pretty big thing to ask for. Why is this necessary?

Reply:
This is something I noticed is needed at the moment, so I documented it. There's an issue for it at https://github.com/Automattic/wp-openid-connect-server/issues/19, once that is fixed these instructions can be removed.